src/HyperFormula.ts

```diff
@@ -4046,14 +4048,9 @@ export class HyperFormula implements TypedEmitter {
    */
   public validateFormula(formulaString: string): boolean {
     validateArgToType(formulaString, 'string', 'formulaString')
-    const {ast} = this.extractTemporaryFormula(formulaString)
-    if (ast === undefined) {
-      return false
-    }
-    if (ast.type === AstNodeType.ERROR && !ast.error) {
-      return false
-    }
-    return true
```

Comment:
It is up to you, but I find the previous version more readable. It is mostly a matter of personal preference, though I am not the one making that statement:
- https://gomakethings.com/readability-is-more-important-than-brevity/
- https://dave.cheney.net/paste/clear-is-better-than-clever.pdf

Reply:
You're right. My IDE suggested this change and I clicked `Accept` without thinking about it. Bringing back the previous version.